extensions/cornerstone-dicom-sr/package.json

```diff
@@ -44,9 +44,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.20.13",
-    "@cornerstonejs/adapters": "^1.10.3",
-    "@cornerstonejs/core": "^1.10.3",
-    "@cornerstonejs/tools": "^1.10.3",
+    "@cornerstonejs/adapters": "^1.11.1",
```

Comment:
Could we get the prefetch fixes in as well?  That issue is also related to how the memory gets handled, and I'd like to see the two of them combined.